T/Typst/build_tarballs.jl

```diff
@@ -0,0 +1,36 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Typst"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/typst/typst.git", "b3faef4b80a674294091066e20501e3a5d0f6103")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/typst
+cargo build -p typst-cli --release
+mkdir -p $bindir
+mv target/$rust_target/release/typst $bindir/
+install_license LICENSE
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = filter(!Sys.iswindows, supported_platforms())
```

Comment:
What's the problem with 64-bit windows?

Reply:
See https://buildkite.com/julialang/yggdrasil/builds/2119#0187561d-fa2f-458c-b0a5-50eb1791a347

Reply:
Yes, we definitely need to improve this.

Reply:
> This is probably not your fault, it’s probably a bug in BB.

I think this _is_ @lucifer1004's fault, and _not_ a bug in BB:
```console
sandbox:${WORKSPACE}/srcdir/typst # ls target/${rust_target}/release/typst*
target/x86_64-pc-windows-gnu/release/typst.d    target/x86_64-pc-windows-gnu/release/typst.exe
```
The file to install should have been `target/${rust_target}/release/typst${exeext}`, not `target/${rust_target}/release/typst`